src/common/components/RRVBreadcrumbs/Breadcrumbs.tsx

```diff
@@ -1,7 +1,21 @@
+import React from 'react';
+
 import { Breadcrumbs, Button, Chip, Typography } from '@mui/material';
 import { emphasize, styled } from '@mui/material/styles';
 import { Link } from 'react-router-dom';
 
+interface Crumb {
+  name: string;
+  route: string;
+  primaryIcon?: React.ElementType;
+  secondaryIcon?: React.ElementType;
```

Comment:
Let's be consistent and always use one form. Instead of `React.ElementType`, let's use `React.ReactNode`